bigint/src/bigint.rs

```diff
@@ -101,6 +101,17 @@ impl serde::Deserialize for Sign {
     }
 }
 
+impl fmt::Display for Sign {
+	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		use Sign::*;
+		match *self {
+			Plus => write!(f, "+"),
+			NoSign => write!(f, ""),
+			Minus => write!(f, "-")
+		}
```

Comment:
I think using [`Formatter::pad(&mut self, s: &str)`](https://doc.rust-lang.org/stable/std/fmt/struct.Formatter.html#method.pad) will be more flexible than `write!`.

Reply:
OK :)